### PR TITLE
drone has resin jelly but on a much higher CD

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
35 second c/d, it has jelly
compared to 20 for hivelord.

## Why It's Good For The Game
Gives the hive the ability to deny fire in more comps without hivelard, but hivelard is still way better at it.

## Changelog
:cl:
balance: drone has resin jelly, 35 second cooldown
/:cl:

### CODE REVIEW

**Comment by lKiseki** [comment]
me likey

**Comment by InterroLouis** [comment]
Why not put it into the general xeno abilities list and label it as create_jelly/slow instead so a salty person can give it to everyone in a funni PR?

**Comment by Al-1ce** [comment]
> Why not put it into the general xeno abilities list and label it as create_jelly/slow instead so a salty person can give it to everyone in a funni PR?

Bad idea. It's a support tool and should be used by support castes, otherwise there'd be little point.

File: code/modules/mob/living/carbon/xenomorph/castes/drone/abilities_drone.dm [hunk 1]
```diff
@@ -52,3 +52,8 @@
 	adjustBruteLoss(-amount)
 	adjustFireLoss(-remainder, updating_health = TRUE)
 	adjust_sunder(-amount/20)
+
+// Jelly for drones, has a higher CD than Hivelord.
+
+/datum/action/xeno_action/create_jelly/drone
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/castes/drone/abilities_drone.dm:58]:
```suggestion
/datum/action/xeno_action/create_jelly/slow
```

File: code/modules/mob/living/carbon/xenomorph/castes/drone/castedatum_drone.dm [hunk 2]
```diff
@@ -59,6 +59,7 @@
 		/datum/action/xeno_action/activable/transfer_plasma/drone,
 		/datum/action/xeno_action/activable/corrosive_acid/drone,
 		/datum/action/xeno_action/activable/larval_growth_sting,
+		/datum/action/xeno_action/create_jelly/drone,
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/castes/drone/castedatum_drone.dm:62]:
```suggestion
		/datum/action/xeno_action/create_jelly/slow,
```

**Review by BraveMole — CHANGES_REQUESTED** [review]
skel is right

**Comment by TiviPlus** [comment]
Balance freeze

**Pull request closed without merging**